gino/asyncpg_delegate.py

```diff
@@ -236,7 +247,7 @@ def iterate(self, clause, *multiparams, connection=None,
 
         return rv
 
-    def acquire(self, *, timeout=None, reuse=True):
+    def acquire(self, *, timeout=None, reuse=True, lazy = False):
```

Comment:
unused, should make use of `LazyTransactionalConnection`? (maybe split transaction away from connection?)